src/math-translator.js

```diff
@@ -499,14 +500,17 @@ function getEscapedDecimalSeparator(lang) {
  */
 function getOrderedPairRegexString(lang) {
     const katexColorMacros = `\\\\(?:${KATEX_BASE_COLORS.join('|')})[A-Z]?`;
-    // Assuming single-letter variables and numbers below 1000
-    // (without thousand separator)
+    // Assuming single-letter variables (or \pi) and numbers below 1000
+    // (without thousand separator) or combinations, such as '2\\pi' or '2.1b'
     const integer =
       `-?[0-9]+|-?${katexColorMacros}\\{-?[0-9]+\\}|-?${katexColorMacros}[0-9]`;
-    const variable = `[a-z]|${katexColorMacros}\\{[a-z]\\}`;
+    const variable = `\\\\pi|[a-z]|${katexColorMacros}\\{[a-z]\\}`;
     const decimal = getDecNumberRegexString(lang,
         /* don't include capture groups */ false);
-    const numberOrLetter = `${variable}|${decimal}|${integer}`;
+
+    const numberAndOrLetter =
+        `${variable}|(?:${decimal}|${integer})(?:${variable})?`;
```

Comment:
Looks like this may match with "variable" (or \pi) on both ends. Is that a concern? Should we add a test to assert it's not doing anything strange for such a case?

Reply:
Yeah, that's what I wanted, because we also need to match things like `(2\\pi,-2\\pi)` or `(-0.5a, b)`. I'll think about adding a test to make sure we're not matching too much.